recipes/mosquitto/2.x/conanfile.py

```diff
@@ -0,0 +1,146 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class Mosquitto(ConanFile):
+    name = "mosquitto"
+    license = "EPL-2.0", "EPL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://mosquitto.org"
+    description = """Eclipse Mosquitto MQTT library, broker and more"""
+    topics = ("MQTT", "IoT", "eclipse")
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+                "with_tls": [True, False],
+                "clients": [True, False],
+                "broker": [True, False],
+                "apps": [True, False],
+                "plugins": [True, False],
+                "with_cjson": [True, False],
+            }
+    default_options = {"shared": False,
+                        "with_tls": True,
+                        "clients": False,
+                        "broker": False,
+                        "apps": False,
+                        "plugins": False,  # TODO, there is some logic, just enabling plugin does not work, needs also something else
+                        "with_cjson": False ,
+    }
+
+    _cmake = None
+
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.options.with_tls:
+            self.requires("openssl/1.1.1i")
+        if self.options.with_cjson:
+            self.requires("cjson/1.7.14")
+
+    def configure(self):
+        ...
+        if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
```

Comment:
MSVC MT  should now magically work.
So these lines can be removed.

Reply:
> is nonsense , why ?
> apropos errors, it would really help if all the old reports would be deleted .....

yes, it's non-sense, probably another infra-structure error. that's why it's still EAP :)